packages/events/src/storage/postgres/events/schema/app/ActionType.ts

```diff
@@ -14,6 +14,7 @@ type ActionType =
   | 'ARCHIVE'
   | 'PRINT_CERTIFICATE'
   | 'REQUEST_CORRECTION'
+  | 'REVIEW_CORRECTION_REQUEST'
```

Comment:
I'm wondering if there would be a way to do this with just the `APPROVE_CORRECTION` and `REJECT_CORRECTION` actions, without defining this `REVIEW_CORRECTION_REQUEST` action at all. What do you think?

Reply:
Yeah, we definitely could in my opinion. But the ticket had task regarding `review-correction` specifically which is probably something we want in the future. (We have some discussions in [here](https://github.com/opencrvs/opencrvs-core/issues/9301#issuecomment-2862568799) )

Reply:
Made some changes regarding this. To not include `REVIEW_CORRECTION_REQUEST` in `ActionTypes` directly